src/lib/manualSoftNavigation.ts

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+export const onManualSoftNavigation = (cb: () => void) => {
+  document.addEventListener('reset-web-vital-metrics', cb);
+};
```

Comment:
I  could also easily export something like

```js
export const triggerManualSoftNavigation = () => {
  document.dispatchEvent(new Event('reset-web-vital-metrics'));
}
```

but I'm not sure how you feel about exposing API besides `onXXX` from this package.